### PR TITLE
vagrant: add script to get kubeconfig for vagrant cluster

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

vagrant: add script to get kubeconfig for vagrant cluster

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: vagrant/scripts/get-kubeconfig.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# This script replaces kubeconfig with configuration for vagrant cluster
+
+readonly DIR="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+readonly ROOT_DIR="${DIR}/../../"
+readonly SSH_CONFIG_PATH="${ROOT_DIR}/tmp/ssh-connect.conf"
+readonly VAGRANT_IP="192.168.78.66"
+readonly CONFIG_PATH="${ROOT_DIR}/tmp/config"
+readonly KUBECONFIG_PATH=~/.kube/config
+
+pushd "${ROOT_DIR}"
```
Comment (pmalek-sumo) [inline vagrant/scripts/get-kubeconfig.sh:13]: add a `popd`, ideally in a `trap`

**Review by kkujawa-sumo — APPROVED** [review]

File: vagrant/scripts/get-kubeconfig.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script replaces kubeconfig with configuration for vagrant cluster
+
+readonly DIR="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+readonly ROOT_DIR="${DIR}/../../"
```
Comment (pmalek-sumo) [inline vagrant/scripts/get-kubeconfig.sh:7]: Wouldn't it be better to operate in `/tmp/` instead of littering the repo?

Comment (pmalek-sumo) [reply vagrant/scripts/get-kubeconfig.sh:7]: It may be gitignored but still it adds "one-use" junk to repo dir.

Comment (sumo-drosiek) [reply vagrant/scripts/get-kubeconfig.sh:21]: check it now

File: vagrant/scripts/get-kubeconfig.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	rmdir "${TEMP_DIR}" || true
+	popd || true
+}
+
+err_report() {
+    echo "Script error on line $1"
+	clean_up
+    exit 1
+}
+trap 'err_report $LINENO' ERR
```
Comment (pmalek-sumo) [inline vagrant/scripts/get-kubeconfig.sh:18]:
```suggestion
trap clean_up INT
trap 'err_report $LINENO' ERR
```

?

**Review by pmalek-sumo — APPROVED** [review]

File: vagrant/scripts/get-kubeconfig.sh (hunk 3 above)
Comment (sumo-drosiek) [reply vagrant/scripts/get-kubeconfig.sh:18]: It doesn't work :disappointed:

**Pull request merged**